src/server/acl/acl_family.cc

```diff
@@ -1189,16 +1189,16 @@ void AclFamily::BuildIndexers(RevCommandsIndexStore families) {
   acl::NumberOfFamilies(families.size());
   CommandsRevIndexer(std::move(families));
   CategoryToCommandsIndexStore index;
-  cmd_registry_->Traverse([&](std::string_view name, auto& cid) {
-    auto cat = cid.acl_categories();
+  cmd_registry_->Traverse([&](std::string_view, auto& cid) {
+    const uint32_t cat = cid.acl_categories();
+    const size_t family = cid.GetFamily();
+    const uint64_t bit_index = cid.GetBitIndex();
     for (size_t i = 0; i < 32; ++i) {
-      if (cat & (1 << i)) {
+      if (cat & 1 << i) {
```

Comment:
I always add parenthesis because I always forget the operator precedence :smile: